docs/source/getting-started/tabular-data/agg-and-filter.ipynb

```diff
@@ -0,0 +1,456 @@
+      "text/plain": [
+       "<Figure size 1200x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(12, 6))\n",
+    "sns.lineplot(x=count_year_all[\"Year-Quarter\"].to_list(), y=count_year_all['Actual Count'].to_list(), marker='o', label='Actual Count')\n",
+    "sns.lineplot(x=count_year_all[\"Year-Quarter\"].to_list(), y=count_year_all['DP Count'].to_list(), marker='o', label='DP Count')\n",
+    "\n",
+    "plt.title('Actual Count vs DP Count Over Years and Quarters')\n",
+    "plt.xlabel('Year-Quarter')\n",
+    "plt.ylabel('Count')\n",
+    "plt.xticks(rotation=45) \n",
+    "plt.legend()\n",
+    "\n",
+    "plt.grid(True)\n",
+    "plt.tight_layout()  \n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While the differentially private counts still follow the same trend as the actual values, there is more variance because of the smaller sizes of the year-quarter groups. \n",
+    "\n",
+    "### Fine-grained grouping adds more noise\n",
+    "\n",
+    "Larger datasets require less noise to be added to preserve privacy. Grouping by multiple keys can lead to smaller partitions, and adding noise to the smaller partitions may lower the utility of your results. \n",
```

Comment:
This depends on the query. Count queries require a constant amount of noise relative to the dataset size. Grouping by multiple keys can lead to adding the same quantity of noise more times.